src/core/Cline.ts

```diff
@@ -1511,6 +1518,20 @@ export class Cline {
 								if (!didApprove) {
 									break
 								}
+								if (
+									command.includes("&gt;") ||
+									command.includes("&lt;") ||
+									command.includes("&quot;") ||
+									command.includes("&amp;") ||
+									command.includes("&apos;")
```

Comment:
same here, can use the previously suggested `isUnallowedHtmlEntities(..)` function